apps/block_scout_web/lib/block_scout_web/controllers/address_celo_controller.ex

```diff
@@ -26,7 +30,12 @@ defmodule BlockScoutWeb.AddressCeloController do
         coin_balance_status: CoinBalanceOnDemand.trigger_fetch(address),
         exchange_rate: Market.get_exchange_rate(Explorer.coin()) || Token.null(),
         counters_path: address_path(conn, :address_counters, %{"id" => address_hash_string}),
-        tags: get_address_tags(address_hash, current_user(conn))
+        tags: get_address_tags(address_hash, current_user(conn)),
+        validator_or_group_sum: validator_or_group_sum,
```

Comment:
I feel it would be clearer if we wrapped this in a map of some kind and passed that to the template as a single param `celo_epoch` (or similar named). 

As it stands it seems we are 

* adding an extra 5 parameters to ~23 templates
* destructuring the result of `EpochUtil.get_address_summary` just to repackage it as discrete named parameters for each template

This will be tough to maintain as upstream changes add and change templated values - i.e. what are celo and what are upstream changes? 

Additionally it's not clear why `EpochUtil.get_address_summary` has to return a tuple of ordered values - what these values are is not clear from the POV of the calling function, a caller has to be aware of the semantics of the return value and the index of each value. 

I would recommend returning a map `%{voting_sum: voting_sum, ...}` from the `EpochUtils` function and then passing this to `render` with a call like `celo_epoch_params: address_summary` to reduce the amount of copy + paste here.